Makefile

```diff
@@ -270,27 +257,8 @@ endif
 validate-manifests: generate manifests check-missing-files
 
 .PHONY: kustomize
-KUSTOMIZE = $(shell pwd)/bin/kustomize
 kustomize: ## Download kustomize locally if necessary
```

Comment:
Please remove comment about downloading kustomize is necessary, Makefile is no longer doing that.

I think maybe the whole target can go away?

BTW note that comments starting with `## ...` are specially important as they make documentation you can read when you run just with `make`